yaml/pois.yaml

```diff
@@ -926,11 +926,24 @@ filters:
       kind: playground
       tier: 6
   # school
-  - filter: {amenity: school}
-    min_zoom: { min: [ { max: [ { sum: [ { col: zoom }, 2.3 ] }, *tier6_min_zoom ] }, 15 ] }
+  - filter:
+      amenity:
+        - school
+        - kindergarten
+    min_zoom:
+      lookup:
+        key: { col: way_area }
+        op: '>='
+        table:
+          - [ 13,  100000 ]
+          - [ 14,   50000 ]
+          - [ 15,   10000 ]
+          - [ 16,    5000 ]
+          - [ 17,    2000 ]
+        default: 18
```

Comment:
Our SemVer says we can only change a full zoom default at a time to be minor version increment. So let's make the default zoom 17 and remove the zoom 17 area filter?

Reply:
Resolved via https://github.com/tilezen/vector-datasource/pull/1764/commits/5b1002d33bb145dde6de984b5638db53ae83d465.